jest.config.js

```diff
@@ -49,6 +51,7 @@ module.exports = {
     '^.+\\.[jt]sx?$': '<rootDir>/scripts/utils/jest-transform-js.js',
     '^.+\\.mdx$': '@storybook/addon-docs/jest-transform-mdx',
   },
+  transformIgnorePatterns: [`<rootDir>/node_modules/(?!${esModules})`],
```

Comment:
without this jest will end up having issues with those packages

Reply:
This pattern assumes they'll always get hoisted which isn't guaranteed to happen

Reply:
Yes right. Want me to remove rootdir ?

`/node_modules/(?!${esModules})`